### PR TITLE
Ana/fix showAmounts logic in ParticipantList

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Literally one line change:

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f64c1c32168075120756dcf--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
Please, leave this at it is:

```
.participant div {
  display: flex;
  align-items: center;
  width: 7rem;
}
``` 

The difference is:

- Without width:

image


- With width:

image

**Comment by Bitcoinera** [comment]
Self-approve :smiley_cat:

File: app/src/components/governance/ParticipantList.vue [hunk 1]
```diff
@@ -141,6 +141,7 @@ h4 {
 .participant div {
   display: flex;
   align-items: center;
+  width: 7rem;
```
Comment (jbibla) [inline app/src/components/governance/ParticipantList.vue:144]: this isn't really going to work on mobile

Comment (Bitcoinera) [reply app/src/components/governance/ParticipantList.vue:144]: some other idea how to accomplish the straight columns otherwise?

Comment (jbibla) [reply app/src/components/governance/ParticipantList.vue:144]: i guess setting min and max widths with width being 100%

Comment (jbibla) [reply app/src/components/governance/ParticipantList.vue:144]: lol https://github.com/luniehq/lunie/issues/4941

File: app/src/components/governance/PageProposal.vue [hunk 2]
```diff
@@ -10,14 +10,18 @@
       />
 
       <ProposalStatusBar
-        v-if="tallyHasValues"
+        v-if="tallyHasValues && !isPolkadotDemocracy"
```
Comment (jbibla) [inline app/src/components/governance/PageProposal.vue:13]: let's unhide it and just make sure the bar shows 100% if it's 100% or over

Comment (faboweb) [reply app/src/components/governance/PageProposal.vue]: why? imagine kickstarter. 4000% funded

Comment (Bitcoinera) [reply app/src/components/governance/PageProposal.vue]: I obeyed xD

Comment (jbibla) [reply app/src/components/governance/PageProposal.vue]: how do you show 400% on a bar? the bar can only be 100% full - so show it as 100% full and say 400% !!

Comment (faboweb) [reply app/src/components/governance/PageProposal.vue]: was it problem before that the bar was 400% width?

Comment (jbibla) [reply app/src/components/governance/PageProposal.vue]: no i think it was fine before - now we'll unhide it and all is well. nothing to worry about here!

**Comment by jbibla** [comment]
i think we can close those or remove all the pieces except the showAmount logic reversal

**Review by jbibla — APPROVED** [review]

**Pull request merged**